build.gradle

```diff
@@ -37,7 +37,7 @@ allprojects {
         servoVersion = '0.12.21'
         governatorVersion = '1.17.5'
         archaiusVersion = '0.7.6'
-        jacksonVersion = '2.9.4'
+        jacksonVersion = '2.10.3'
```

Comment:
Seems like `2.9.10` include the same security fixes, I don't mind jumping further but would probably want it separately to make sure nothing breaks (it's 2 years worth of changes).

Reply:
Thanks for the fast reply!

I saw 2 vulnerabilities in 2.9.10:
https://github.com/advisories/GHSA-fmmc-742q-jg75 
https://github.com/advisories/GHSA-mx7p-6679-8g3q

If you prefer to do it in steps, maybe we should upgrade to 2.9.10.1?
(Maybe there are more in later releases but these are the ones found in a quick search)

Reply:
Let's do it in 2 steps if you don't mind, the latest 2.9.x and then try out `2.10.x`?

Reply:
`2.9.10.3` seems like a safe bet?

Reply:
Sounds Good :)

I'll upgrade to 2.9.10.3 first.

Thanks! :)